hack/gen-bootstrap-bindata.sh

```diff
@@ -18,11 +18,14 @@ popd > /dev/null
 
 pushd "${OS_ROOT}" > /dev/null
   "${GOPATH}/bin/go-bindata" -nocompress -nometadata -prefix "bootstrap" -pkg "bootstrap" \
-                                   -o "${OUTPUT_PARENT}/pkg/bootstrap/bindata.go" -ignore "README.md" \
-                                   ${EXAMPLES}/image-streams/... \
-                                   ${EXAMPLES}/db-templates/... \
-                                   ${EXAMPLES}/jenkins/pipeline/... \
-                                   ${EXAMPLES}/quickstarts/...
+    -o "${OUTPUT_PARENT}/pkg/bootstrap/bindata.go" \
+    -ignore "README.md" \
+    -ignore application-template.json \
+    ${EXAMPLES}/image-streams/... \
+    ${EXAMPLES}/db-templates/... \
+    ${EXAMPLES}/jenkins \
+    ${EXAMPLES}/jenkins/pipeline \
+    ${EXAMPLES}/quickstarts/...
```

Comment:
lol.  and @stevekuznetsov sure, I thought I was already improving it, but i can go the rest of the way, just for you.